apps/docs/app/ui/components/componentExample/ComponentPreviewWrapper.tsx

```diff
@@ -44,11 +44,11 @@ const ComponentPreviewWrapper = memo(({ preview, toggleButton, minHeight = "13re
                     colorMode={localColorMode}
                 />
             </div>
-            <Card className="hd-component-preview-wrapper__card" size="sm" style={{ minHeight: minHeight }}>
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/c2d7e67e-c3b2-4b5a-ab93-c8bca183d33a)
I think there's an issue here, my components have way less props shown.

Reply:
ya I already mentioned this to Franck. it'll be fixed later on.